src/routes/about/+page.svelte

```diff
@@ -1 +1,260 @@
-<p>about page. under construction...</p>
+<script>
+	import { onMount } from 'svelte';
+
+	let eboard_members = [
```

Comment:
Update this list

Reply:
Carson would also like a spot as a "Founding President" so be sure to add that as well

https://discord.com/channels/1088161921666142290/1088161922416910493/1266583882988261458